src/main/java/com/sprint/mission/discodeit/controller/ApiController.java

```diff
@@ -0,0 +1,43 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.UserDto;
+import com.sprint.mission.discodeit.entity.BinaryContent;
+import com.sprint.mission.discodeit.mapper.BinaryContentMapper;
+import com.sprint.mission.discodeit.mapper.UserMapper;
+import com.sprint.mission.discodeit.service.BinaryContentService;
+import com.sprint.mission.discodeit.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@Controller
+@RequiredArgsConstructor
+public class ApiController {
+    private final UserService userService;
+    private final UserMapper userMapper;
+    private final BinaryContentService binaryContentService;
+    private final BinaryContentMapper binaryContentMapper;
+
+    @RequestMapping("/")
+    public String index() {
+        return "user-list.html";
+    }
+
+    @ResponseBody
+    @RequestMapping(method = RequestMethod.GET, value = "/api/user/findAll")
```

Comment:
```suggestion
    @RequestMapping(method = RequestMethod.GET, value = "/api/users")
```
위와같이 restful하게 설꼐 하시기 바랍니다.

Reply:
아래 다른 url도 최대한 restful하게 설계하시는것을 추천드립니다.